cli/pkg/workspace/template/copy.go

```diff
@@ -0,0 +1,81 @@
+package template
+
+import (
+	"embed"
+	"github.com/Masterminds/sprig"
+	"html/template"
+	"io/fs"
+	"os"
+	"path/filepath"
+)
+
+type Variables struct {
+	NgrokEnabled            bool
+	Version                 string
+	Namespace               string
+	Host                    string
+	LoadBalancerAnnotations string
+}
+
+//go:embed src
+var templateDir embed.FS
+
+func CopyToDir(path string, data Variables) error {
+	entries, err := templateDir.ReadDir("src")
+	if err != nil {
+		return err
+	}
+
+	for _, entry := range entries {
+		if err := recCopy(path, "src", entry, data); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// Recursively copies and templates entries to the workspace
+func recCopy(writePath string, templatePath string, entry fs.DirEntry, data Variables) error {
+	dstPath := filepath.Join(writePath, entry.Name())
+	templatePath = filepath.Join(templatePath, entry.Name())
+
+	if !entry.IsDir() {
+		content, err := templateDir.ReadFile(templatePath)
+		if err != nil {
+			return err
+		}
+		if _, err := os.Stat(dstPath); err == nil {
```

Comment:
This was a bug: The `create` command was previously overwriting config that was already present.